LibreNMS/Alert/Transport/Api.php

```diff
@@ -100,7 +114,7 @@ public static function configTemplate()
             ],
             'validation' => [
                 'api-method' => 'in:GET,POST',
-                'api-url' => 'required|url'
+                'api-url' => 'required|string' //we need to allow special chars & spaces if GET
```

Comment:
We need to change this cause api-url would not pass validation with spaces in it.

Reply:
solved as well